### DISCUSSION TITLE
ESP-EYE

### DISCUSSION THREAD

**Discussion by Shek20** [micropython/micropython#14504] [Core Development]
I've been working with micropython on the ESP32-C3, but now I want to switch to using the ESP-EYE board. I need to know if it's feasible to utilize  micropython on the ESP-EYE and whether this has been accomplished before.

**Comment by scruss** [comment]
Can't say for sure, but looking at the specs from [ESP-EYE Development Board V2.1 : ID 4095 : Adafruit Industries](https://www.adafruit.com/product/4095):

> *    WiSoC ESP32 dual core Tensilica LX6 processor with WiFi and Bluetooth
> *    Memory: 8MB PSRAM
> *    Storage: 4MB Flash
> *    Camera: 2MP OV2640 Camera
> *    Audio: MEMS Microphone
> *    USB: 1 microUSB port for power and programming over serial
> *    Misc: Reset, boot and function buttons, 2x LEDs
> *    Dimensions: 41 x 21 mm

that's a solid "probably works". You'll likely need the [Support for SPIRAM / WROVER](https://micropython.org/download/ESP32_GENERIC/) firmware version.

@shariltumin has done great work with ESP32 camera boards (such as [shariltumin / esp32-cam-micropython-2022](https://github.com/shariltumin/esp32-cam-micropython-2022)) but I don't see the ESP-EYE in there. Adafruit claim it's discontinued: can you still get these?
